### DISCUSSION TITLE
Planetscale works fine except 1 query

### DISCUSSION THREAD

**Discussion by iiDon** [vercel/community#4368] [Help]
### Summary

Vercel can't read a specific query of Planetscale after changing value manully from the database directly.

### Example

http://test.zan-azyan.com/

### Steps to Reproduce

Hello Everyone!

I deployed my Nextjs + planetscale project 3-5 Monthes ago and it works fine, Unitl I change a field value directly from planetscale itself.

After that when i try to fetch that table i got loading forever without any error.

What I tried?

1- Loged the value i got from backend, it was undefiend
2- Tried to add on that table by the system itself and it worked successfully.
3- Tried to test that on localhost and fetched that table data without any issue or error.
4- Tried to deploy the system with VPS and worked fine.
5- Restored the backup database but nothing changed.
6- Deployed the system again on vercel but nothing changed.
7- Deleted the DATABASE_URL from Enviroment Varibales then added again.
8- Redeployed the system.

 Every Query is working on the whole system except the Query of GET data for the specific table i have changed the value of.

Can anyone help me please?

**Comment by NathanVG** [comment]
I am having the same issue since yesterday. 
I am using t3 stack with trpc and prisma
Locally all my calls work fine, but calls to one specific table are no longer working on my deployed version since yesterday. 
When I check the network tab it looks like the call is not actually resolving, it just keeps loading forever.

**Reply by iiDon** [reply]
Yes! Im using the same stack, T3 Stack trpc and prisma. 

DId you change any value of that table manually? or the call just broke suddenly without doing anything?

**Reply by NathanVG** [reply]
So we're copying a production db every night and then running some calculations to add extra fields on it with python. Not really a manual change, but it is an external change. It worked perfectly up until yesterday.

**Reply by NathanVG** [reply]
and it's all queries to a specific table, all other queries to other tables are working just fine.

**Comment by NathanVG** [comment]
@iiDon I just checked out your example and seems like you have the exact same issue as me, could you do me a favour and check if you still have the same issue if you use a different query for the same table, for example getHomePagePlanById?

<img width="673" alt="image" src="https://github.com/vercel/community/assets/3620183/3d071181-a6cc-470c-b863-b55ec29fb835">

for me it looks the same with the call not getting a status code in the network tab and seemingly be stuck in loading forever. 

I also opened an issue with Prisma (https://github.com/prisma/prisma/issues/21368) but it seems to be either a Vercel or Planetscale issue

**Comment by NathanVG** [comment]
This issue just resolved itself for me. I still have a support case running with vercel so I will follow up on what the cause was. 

Judging from your test url it also resolved itself for you @iiDon.

**Reply by iiDon** [reply]
Sorry for late answer, but i faced the same thing. It fixed without doing anything! 

I want edit again on the database but im scared that i get the same issue again
